### PR TITLE
Expand smtp-open-relay resources into vulns

### PR DESCRIPTION
**Pull request by jesusfcr**
Expand resources into vulnerabilities

Validated against:
```sh
docker run -p 587:25 -p 25:25 -d -e POSTFIX_myhostname=smtp.domain.tld mwader/postfix-relay
```

### CODE REVIEW

File: cmd/vulcan-smtp-open-relay/main.go [hunk 1]
```diff
@@ -62,46 +63,39 @@ var (
...
-				networkResource := map[string]string{
-					"Hostname": target,
-					"Port":     strconv.Itoa(port.PortId),
-				}
-				gr.Rows = append(gr.Rows, networkResource)
+				vuln := openRelay
+				vuln.AffectedResource = fmt.Sprintf("%d/%s", port.PortId, port.Protocol)
+				vuln.Fingerprint = helpers.ComputeFingerprint()
+				vuln.Labels = []string{"issue", "smtp", "discovery"}
+				vuln.Resources = []report.ResourcesGroup{{
```
Comment (jesusfcr) [inline cmd/vulcan-smtp-open-relay/main.go:82]: Do we need to keep this? as it is redundant.

**Comment by jesusfcr** [comment]
Should we set as critical?

File: cmd/vulcan-smtp-open-relay/main.go (hunk 1 above)
Comment (julianvilas) [reply cmd/vulcan-smtp-open-relay/main.go:82]: You mean the `Resources` right? If so I totally agree. Kill it with fire!

**Comment by julianvilas** [comment]
> Should we set as critical?

That is a good question. I'm checking for example [how Tenable reports it](https://www.tenable.com/plugins/nessus/10262) and they report as `Critical` for `CVSS v2` and `High` for `CVSS v3`. I'm a bit more on the side of reporting it as `High` because the exploitation impact is limited. For me `Critical` means full compromise.

**Review by julianvilas — APPROVED** [review]
LGTM!

File: cmd/vulcan-smtp-open-relay/main.go (hunk 1 above)
Comment (jesusfcr) [reply cmd/vulcan-smtp-open-relay/main.go]: Done

**Review by julianvilas — APPROVED** [review]

**Pull request merged**